module-1/README.md

```diff
@@ -63,55 +117,255 @@ export const environment = {
+
+The standard structure of an AWS CDK application consists of a `bin` folder and a `lib` folder.
+
+* The `bin` folder is where we will define the entry point for the CDK application.
+* The `lib` folder is where we will define all our workshop infrastructure components.
+
+before we start implementing our code, let's get typescript to watch for file changes and compile the javascript whenever a change is detected by running the following command from the terminal.
+
+```bash
+npm run watch
+```
 
-* Install the AWS Amplify CLI:
-    * npm install -g @aws-amplify/cli
-    * amplify configure
-* Navigate to the `frontend` directory.
-* Run `amplify init` in the `frontend` directory and select the following options:
-    * Choose your default editor: `Visual Studio Code`
-    * Choose the type of app that you're building: `javascript`
-    * What javascript framework are you using: `angular`
-    * Source Directory Path: `src`
-    * Distribution Directory Path: `dist`
-    * Build Command: `npm run build -- --prod`
-    * Start Command: `npm start`
-* Run `amplify add hosting`
-    * Select `PROD (S3 with CloudFront using HTTPS)`
-    * Provide a [globally unique name](https://docs.aws.amazon.com/AmazonS3/latest/dev/BucketRestrictions.html) for your hosting bucket
-* Run `amplify publish`
+#### Module 1 Files
+
+Next, Within the lib folder you should find a file with the default name of `cdk-stack.ts`.  Rename this file to `Module1Stack.ts`.
+
+Open the file `Module1Stack.ts` in VS Code and ename the class to `Module1Stack`, as illustrated below.
+
+```typescript
+import cdk = require('@aws-cdk/cdk');
+
+export class Module1Stack extends cdk.Stack {
+  constructor(scope: cdk.Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    // The code that defines your stack goes here
+  }
+}
+```
+
+Following the change of filename and classname, you should now update the references in the `cdk.ts` file, as such.
+
+```typescript
+#!/usr/bin/env node
+import 'source-map-support/register';
+import cdk = require('@aws-cdk/cdk');
+import { Module1Stack } from '../lib/Module1Stack';
+
+const app = new cdk.App();
+new Module1Stack(app, 'Module1');
+```
+
+Now let's go through defining the S3 and CloudFront infrastructure.
+
+#### Define the S3 bucket
+
+First of all, we are going to define our S3 bucket and define the web index document as 'index.html'
+
+```typescript
+const projectName = "mythical-mysfits";
+const bucket = new s3.Bucket(this, "Bucket", {
+  bucketName: projectName + "-frontend-" + cdk.Aws.accountId,
+  websiteIndexDocument: "index.html"
+});
+```
+
+#### Restricted access to the S3 bucket
+
+Next we obtain the cloudfront origin access identity so that the s3 bucket may be restricted to it.  Then we define an IAM policy to restrict access to the S3 bucket so that only CloudFront distribution can access it.
+
+```typescript
+const origin = new cloudfront.CfnCloudFrontOriginAccessIdentity(this, "BucketOrigin", {
```

Comment:
Don't we have an L2 for this? Should we create one? I rather the lab will use L2s as much as possible.

Reply:
There doesn't appear to be.  I will research this.